examples/python/e50hx.py

```diff
@@ -69,30 +70,30 @@ def exitHandler():
     # sensor.setDebug(True);
 
     # output the serial number and firmware revision
-    print
-    print "Device Description:", sensor.getDeviceDescription()
-    print "Device Location:", sensor.getDeviceLocation()
-    print
+    print()
+    print("Device Description:", sensor.getDeviceDescription())
+    print("Device Location:", sensor.getDeviceLocation())
+    print()
 
     # update and print available values every second
     while (1):
-        print "System Voltage:",
-        print sensor.getAnalogValue(sensorObj.E50HX.AV_System_Voltage),
-        print " ",
-        print sensor.getAnalogValueUnits(sensorObj.E50HX.AV_System_Voltage)
+        print("System Voltage:", end=' ')
+        print(sensor.getAnalogValue(sensorObj.E50HX.AV_System_Voltage), end=' ')
+        print(" ", end=' ')
```

Comment:
Ugh, the snippet I'm pasting in has spaces between the 10.0 values, not sure why it doesn't print like that in github